WordPress/Classes/Utility/PushNotificationsManager.swift

```diff
@@ -59,41 +52,30 @@ final public class PushNotificationsManager : NSObject
     
     // MARK: - Public Methods
     
-    /**
-     *  @brief      Registers the device for Remote + User Notifications.
-     *  @details    We'll wire Badge + Sound + Alert notifications, along with support for
-     *              iOS User Notifications Actions.
-     */
-    func registerForPushNotifications() {
+    
+    /// Registers the device for Remote Notifications: Badge + Sounds + Alerts
+    ///
+    func registerForRemoteNotifications() {
         if sharedApplication.isRunningSimulator() || sharedApplication.isAlphaBuild() {
             return;
         }
         
-        // Remote Notifications Registration
         sharedApplication.registerForRemoteNotifications()
-
-        // User Notifications Registration
-        let categories  = interactiveHandler.supportedNotificationCategories()
-        let settings    = UIUserNotificationSettings(forTypes: [.Badge, .Sound, .Alert], categories: categories)
-        sharedApplication.registerUserNotificationSettings(settings)
     }
     
 
     
-    /**
-     *  @details    Indicates whether Push Notifications are enabled in Settings.app, or not.
-     */
+    /// Indicates whether Push Notifications are enabled in Settings.app, or not.
     func notificationsEnabledInDeviceSettings() -> Bool {
         return (sharedApplication.currentUserNotificationSettings()?.types ?? .None) != .None
     }
     
     
     
-    /**
-     *  @brief      Registers the Device Token agains WordPress.com backend, if there's a default account.
-     *  @details    Both Helpshift and Mixpanel will also be initialized. The token will be persisted across
-     *              App Sessions.
-     */
+    /// Registers the Device Token agains WordPress.com backend, if there's a default account.
+    ///
+    /// - Note: Both Helpshift and Mixpanel will also be initialized. The token will be persisted across App Sessions.
+    ///
     func registerDeviceToken(tokenData: NSData) {
         // We want to register Helpshift regardless so that way if a user isn't logged in
         // they can still get push notifications that we replied to their support ticket.
```

Comment:
This comment is for the two lines of code just below this one.
```
Helpshift.sharedInstance().registerDeviceToken(tokenData)
Mixpanel.sharedInstance().people?.addPushDeviceToken(tokenData)
```
We chatted briefly about this in Slack, just documenting here for posterity.  In both cases, `sharedInstance()` can return `nil` if the app was built without the necessary keys in .wpcom_app_credentials.  The result is the app will probably crash for most contributors with an  “unexpectedly found nil when unwrapping an optional” error.  

I know its not strictly a part of this PR, but maybe we could add some nil checks?  A separate PR would also be fine. :)